Library/Formula/terraform.rb

```diff
@@ -145,14 +63,12 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
+      terraform_files = `go list ./... | grep -v /vendor/`.strip.gsub("\n", " ")
+      system "go test #{terraform_files}"
```

Comment:
We prefer commands to be joined with `"example", "example2"` rather than unified in-formula. Is that not working here?
```ruby
system "go", "test", terraform_files
```
And below:
```ruby
system "gox", "-arch", arch, "-os", "darwin", "-output", "bin/terraform-{{.Dir}}", terraform_files
```